index.js

```diff
@@ -199,6 +200,14 @@ class CsvParser extends Transform {
   }
 
   _transform (data, enc, cb) {
+    // is it UTF8?
+    if (isUtf8(data)) {
```

Comment:
since `is-utf` uses [its own loop](https://github.com/wayfind/is-utf8/blob/master/is-utf8.js#L5) for iterating over the bytes passed to it, I'd like to see how this impacts performance before signing off on it.

Reply:
Not mutch. And it can also be just a little sniff, it dosent need entire file.

```
isUtf8(data.slice(3,7))
```